src/components/country/Country.js

```diff
@@ -0,0 +1,77 @@
+import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+import DisplayCountries from '../countries/DisplayCountries';
+import './country.css';
+
+const Country = ({
```

Comment:
- [ ] **[OPTIONAL]** It would be nice to add a'Back' button to the **`Country`** page as well.